src/main/java/com/iexec/core/feign/ResultRepoClientWrapper.java

```diff
@@ -0,0 +1,51 @@
+package com.iexec.core.feign;
+
+import java.util.Optional;
+
+import com.iexec.common.result.eip712.Eip712Challenge;
+import com.iexec.core.chain.ChainConfig;
+
+import org.springframework.retry.annotation.Recover;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+@Service
+public class ResultRepoClientWrapper {
```

Comment:
Should we call it ResultRepoService instead and move it to the correct package?

Reply:
Clearer yes, done it !